rbc/tests/heavydb/test_daytimeinterval.py

```diff
@@ -181,7 +182,7 @@ def test_generate_time_series(heavydb, start, stop, step, order):
 @pytest.mark.parametrize("start, stop, step, error_msg", invalid_inputs)
 def test_generate_series_invalid_inputs(heavydb, start, stop, step, error_msg):
 
-    if heavydb.version[:2] >= (6, 2):
+    if heavydb.version[:2] <= (6, 2):
```

Comment:
```suggestion
    if heavydb.version[:2] < (6, 2):
```
or update the skip message.